source/includes/_authentication.md

```diff
@@ -9,41 +9,141 @@ curl "https://api.handshq.com/v1/authentication" \
   -H "Authorization: bearer [api_token]"
 ```
 
-> In all subsequent examples make sure to replace `[api_token]` with your API key.
+> In all subsequent examples make sure to replace `[api_token]` with your API token
 
-HandsHQ uses API keys to allow access to the API. You can find your API key in the API tab of your settings page
+HandsHQ uses API tokens to allow access to the API. You can find your API token in the API tab of your settings page
 [here.](https://app.handshq.com/settings/api_configuration)
 
 
-HandsHQ expects for the API key to be included in all API requests to the server in a header that looks like the following:
+HandsHQ expects for the API token to be included in all API requests to the server in a header that looks like the following:
 
 `Authorization: bearer [api_token]`
 
 <aside class="notice">
-You must replace <code>[api_token]</code> with your personal API key.
+You must replace <code>[api_token]</code> with your HandsHQ API token and please be sure to keep it safe!
 </aside>
 
 
-If you need to test that you have a valid API key use the following endpoint:
-
-### HTTP Request
-
-`GET https://api.handshq.com/v1/authentication`
-
-### Response
-
-Successful authentication with return a `200` response, otherwise a `401` will be returned when supplied with an invalid API key.
-
-> 200
+## Testing your authentication details
 
 ```json
 {
     "data": {
-        "id": "dc729c6795ab40608f8d1a7e8d768e31",
+        "id": "abcdefg",
         "type": "actor",
         "attributes": {
             "company_name": "Example Company Ltd"
         }
     }
 }
 ```
+
+If you would like to test that your token is valid, the following request is available:
+
+### Request
+
+`GET https://api.handshq.com/v1/authentication`
+
+### Response
+
+- `200` Successful authentication.
+- `401` The token provided was not recognised as being valid.
+
+## Verifying requests made by HandsHQ
+### Verification strategy - Signed Requests
+<!-- Credit to @jasny for putting together a gist with different example implementations in different languages -->
+
+> All examples assume a key of "my_key" and JSON data of {"bar":"foo"}
+
+> Ruby Example:
+
+```ruby
+  require 'openssl'
+  key = "my_key"
```

Comment:
does it need to be clear that the key would actually be the HandsHQ API token here?